core/src/text/textBuffer.cpp

```diff
@@ -248,8 +239,17 @@ bool TextBuffer::addLabel(const TextStyle::Parameters& _params, Label::Transform
         vertices.push_back({{q.x0, q.y1}, {q.s0, q.t1}, {-1.f,  1.f, 0.f}, _params.fill, stroke});
         vertices.push_back({{q.x1, q.y0}, {q.s1, q.t0}, { 1.f, -1.f, 0.f}, _params.fill, stroke});
         vertices.push_back({{q.x1, q.y1}, {q.s1, q.t1}, { 1.f,  1.f, 0.f}, _params.fill, stroke});
+
+        yMin = std::min(yMin, q.y0);
+        bbox.x = std::max(bbox.x, q.x1);
+        bbox.y = std::max(bbox.y, std::abs(yMin - q.y1));
     }
 
+    // Adjust the bounding boxes
+    bbox.y += metrics.descender * nLine;
+    // Approximate the first left side bearing
+    bbox.x += quads[0].x0;
```

Comment:
Hmm, so right now you are shifting the bounding box by left side bearing of the very first quad (of the first line).
But we should be shifting the bounding box by the lowest left side bearing of first quad in all the lines. Should result in a small improvement.